src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/Common.ps1

```diff
@@ -617,4 +638,25 @@ function Get-DNSNameBasedOnEnvironment ()
          return ".sqltest-eg1.mscds.com"
      }
      return ".database.windows.net"
+}
+
+<#
+	.SYNOPSIS
+	Creates the test environment needed to perform the Sql managed instance CRUD tests
+#>
+function Create-ManagedInstanceForTest ($resourceGroup)
+{
+	$managedInstanceName = Get-ManagedInstanceName
+	$credentials = Get-ServerCredential
+	$subnetId = "/subscriptions/ee5ea899-0791-418f-9270-77cd8273794b/resourceGroups/cl_one/providers/Microsoft.Network/virtualNetworks/cl_initial/subnets/CooL"
```

Comment:
This should not be hardcoded. @ziwa-msft implemented the ability to create subnet at test run time.